File: frontend/urls.py
from django.urls import path
from .views import index

urlpatterns = [
    path('', index),
    path('premier-league/', index),
    path('premier-league/fixture-planner/', index),
    path('premier-league/player-ownership/', index),
    path('premier-league/player-statistics/', index),
    path('premier-league/live-fixtures/', index),

    path('premier-league/fdr-planner/', index),
    path('premier-league/rotation-planner/', index),
    path('premier-league/periode-planner/', index),

    path('eliteserien/', index),
    path('eliteserien/fdr-planner/', index),
    path('eliteserien/fdr-planner-team-id/', index),
    path('eliteserien/rotation-planner/', index),
    path('eliteserien/periode-planner/', index),
    path('eliteserien/player-ownership/', index),
    path('eliteserien/search-user-names/', index),
    path('eliteserien/rank-statistics/', index),
    path('eliteserien/live-fixtures/', index),

    path('fixture-planner/', index),
    path('fixture-planner/fdr-planner/', index),
    path('fixture-planner/rotation-planner/', index),
    path('fixture-planner/periode-planner/', index),
    path('fixture-planner-eliteserien/', index),
    path('fixture-planner-eliteserien/rotation-planner/', index),
    path('fixture-planner-eliteserien/fdr-planner/', index),
    path('fixture-planner-eliteserien/periode-planner/', index),
    
    path('statistics/player-ownership/', index),
    path('statistics-premier-league/player-ownership/', index),
    path('statistics/search-user-names/', index),
    path('statistics/rank-statistics/', index),
]

File: src/frontend/constants/urls.ts
export const URLS = {
  INTERNAL: {
    PREMIER_LEAGUE: {
      BASE: 'premier-league',
      FDR_PLANNER: 'premier-league/fdr-planner',
      ROTATION_PLANNER: 'premier-league/rotation-planner',
      FDR_PLANNER_TEAM_ID: 'premier-league/fdr-planner-team-id',
      PERIOD_PLANNER: 'premier-league/periode-planner',
      PLAYER_OWNERSHIP: 'premier-league/player-ownership',
      PLAYER_STATS: 'premier-league/player-statistics',
      LIVE_FIXTURES: 'premier-league/live-fixtures',
      PRICE_CHANGES: 'premier-league/price-change',
    },

    ELITESERIEN: {
      BASE: 'eliteserien',
      FDR_PLANNER: 'eliteserien/fdr-planner',
      ROTATION_PLANNER: 'eliteserien/rotation-planner',
      FDR_PLANNER_TEAM_ID: 'eliteserien/fdr-planner-team-id',
      PERIOD_PLANNER: 'eliteserien/periode-planner',
      PLAYER_OWNERSHIP: 'eliteserien/player-ownership',
      PLAYER_STATS: 'eliteserien/player-statistics',
      SEARCH_USER_NAME: 'eliteserien/search-user-names',
      RANK_STATS: 'eliteserien/rank-statistics',
      LIVE_FIXTURES: 'eliteserien/live-fixtures',
      PRICE_CHANGES: 'eliteserien/price-change',
    },
  },

  API: {
    FDR: {
      PREMIER_LEAGUE_FROM_TEAM_ID: '/fixture-planner/get-fdr-data-from-team-id/',
      ELITESERIEN_FROM_TEAM_ID: 'eliteserien/fixture-planner/fantasy-team-fdr/',
    },
  },

  EXTERNAL: {
    PERSONAL: {
      TWITTER: 'https://twitter.com/Ole_Borge',
      GITHUB: 'https://github.com/olemabo/viewfantasystats',
    },
    OFFICIAL: {
      FPL: 'https://fantasy.premierleague.com/',
    },
    SPREADSHEETS: {
      DAGFINN_THON:
        'https://docs.google.com/spreadsheets/d/10vu3C6lib9qGyoZ6As6qfRZywZUrxpA0zuZogI7D5tA/edit?gid=1349521432#gid=1349521432',
    },
  },
};